docs/how-to-connect/desktop-sdks/electron-app.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Electron App
+description: An Electron app example for Casdoor
+keywords: [electron, sdk]
+---
+An [Electron app example](https://github.com/casdoor/casdoor-electron-example) for Casdoor.
+
+# How to run example
+
+## Init the example
+
+You need to init requires 6 parameters, which are all string type:
+
+| Name                 | Description                                                                                      | Path                   |
+| -------------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| serverUrl            | your Casdoor server URL                                                                          | `src/App.js`         |
+| clientId             | the Client ID of your Casdoor application                                                        | `src/App.js`         |
+| appName              | the name of your Casdoor application                                                             | `src/App.js`         |
+| redirectPath         | the path of the redirect URL for your Casdoor application, will be `/callback` if not provided | `src/App.js`         |
+| clientSecret         | the Client Secret of your Casdoor application                                                   | `src/App.js`         |
+| casdoorServiceDomain | your Casdoor server URL                                                                          | `public/electron.js` |
+
+If you don't set these parameters, this project will use the [Casdoor online demo](https://door.casdoor.com/) as the defult Casdoor server and use the [Casnode](https://door.casdoor.com/applications/app-casnode) as the default Casdoor application.
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm run dev` or `yarn dev`
+
+Builds the electron app and run this app.
+
+### `npm run make` or `yarn make`
+
+Package and distribute your application. It will create the `out` folder where your package will be located:
+`// Example for macOS out/  
+├── out/make/zip/darwin/x64/casdoor-electron-example-darwin-x64-1.0.0.zip  
+├── ...  
+└── out/casdoor-electron-example-darwin-x64/casdoor-electron-example.app/Contents/MacOS/casdoor-electron-example`
+
+## Prview the example
+
+After you run this electron application, a new window will be showed on your desktop.
+![electron login](/img/howto-desktop-electron-login.png "Image not found: /img/howto-desktop-electron-login.png")
```

Comment:
`"Image not found: /img/howto-desktop-electron-login.png"`

what's the use of it ?